web/index.html

```diff
@@ -43,9 +43,268 @@ <h2 class="f5 tc f3-l fw1 mb4 near-white lh-title">Welcome to StackStorm! </h2>
+					<option value="Sierra Leone">Sierra Leone</option>
+					<option value="Singapore">Singapore</option>
+					<option value="Slovakia">Slovakia</option>
+					<option value="Slovenia">Slovenia</option>
+					<option value="Solomon Islands">Solomon Islands</option>
+					<option value="Somalia">Somalia</option>
+					<option value="South Africa">South Africa</option>
+					<option value="Spain">Spain</option>
+					<option value="Sri Lanka">Sri Lanka</option>
+					<option value="Sudan">Sudan</option>
+					<option value="Suriname">Suriname</option>
+					<option value="Swaziland">Swaziland</option>
+					<option value="Sweden">Sweden</option>
+					<option value="Switzerland">Switzerland</option>
+					<option value="Syria">Syria</option>
+					<option value="Tahiti">Tahiti</option>
+					<option value="Taiwan">Taiwan</option>
+					<option value="Tajikistan">Tajikistan</option>
+					<option value="Tanzania">Tanzania</option>
+					<option value="Thailand">Thailand</option>
+					<option value="Togo">Togo</option>
+					<option value="Tokelau">Tokelau</option>
+					<option value="Tonga">Tonga</option>
+					<option value="Trinidad &amp; Tobago">Trinidad &amp; Tobago</option>
+					<option value="Tunisia">Tunisia</option>
+					<option value="Turkey">Turkey</option>
+					<option value="Turkmenistan">Turkmenistan</option>
+					<option value="Turks &amp; Caicos Is">Turks &amp; Caicos Is</option>
+					<option value="Tuvalu">Tuvalu</option>
+					<option value="Uganda">Uganda</option>
+					<option value="Ukraine">Ukraine</option>
+					<option value="United Arab Erimates">United Arab Emirates</option>
+					<option value="United Kingdom">United Kingdom</option>
+					<option value="Uraguay">Uruguay</option>
+					<option value="Uzbekistan">Uzbekistan</option>
+					<option value="Vanuatu">Vanuatu</option>
+					<option value="Vatican City State">Vatican City State</option>
+					<option value="Venezuela">Venezuela</option>
+					<option value="Vietnam">Vietnam</option>
+					<option value="Virgin Islands (Brit)">Virgin Islands (Brit)</option>
+					<option value="Virgin Islands (USA)">Virgin Islands (USA)</option>
+					<option value="Wake Island">Wake Island</option>
+					<option value="Wallis &amp; Futana Is">Wallis &amp; Futana Is</option>
+					<option value="Yemen">Yemen</option>
+					<option value="Zaire">Zaire</option>
+					<option value="Zambia">Zambia</option>
+					<option value="Zimbabwe">Zimbabwe</option>
+					</select>
+                </div>
+                <div class="mt1">
+					<input type="checkbox" name="consent" value="true" id="consent-checkbox">
+					<label for="consent-checkbox">
+      					<span><font color="white">Please check the box if you are happy for StackStorm to keep you updated about our latest products, services and promotions via email.</span>
+   					</label>
                 </div>
               </fieldset>
+              <a href="https://stackstorm.com/privacy-policy//">Privacy Policy</a>
```

Comment:
Minor nitpick - you've got an extra `/` at the end of the URL. Should be single `/` at the end.

Reply:
Thanks. Fixed